src/modules/drawing/DrawingModule.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+    <div v-if="show" class="draw-overlay">
+        <DrawingToolbox
+            :drawing-modes="drawingModes"
+            :current-drawing-mode="currentDrawingMode"
+            @close="hideDrawingOverlay"
+            @setDrawingMode="changeDrawingMode"
+        />
+    </div>
+</template>
```

Comment:
A blank line between template, script and style would be nice. I've seen some Vue files already have this blank line style.